readme.txt

```diff
@@ -321,6 +331,24 @@ Setting surrogate keys for posts with large numbers of taxonomies (such as WooCo
 	}
 	add_filter('pantheon_should_add_terms', 'custom_should_add_terms', 10, 2);
 
+== Other Filters ==
+
+= `pantheon_apc_disable_admin_notices` =

```

Comment:
Does backticks in heading work on wp.org readmes?

Reply:
Yes https://wordpress.org/plugins/pantheon-advanced-page-cache/

Reply:
Wait, in heading maybe not. I guess we'll find out. I couldn't find anything definitive.